src/ray/raylet/node_manager.h

```diff
@@ -187,6 +190,11 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   /// Get the port of the node manager rpc server.
   int GetServerPort() const { return node_manager_server_.GetPort(); }
 
+  void Update(std::shared_ptr<const syncer::RaySyncMessage> message) override;
```

Comment:
should these two functions belong to cluster_task_manager_/cluster_resource_scheduler_ ?

Reply:
Answered here https://github.com/ray-project/ray/pull/23660/files#r850865947